website/content/en/docs/olm-integration/generation.md

```diff
@@ -134,11 +134,13 @@ endif
 PKG_MAN_OPTS ?= $(PKG_CHANNELS) $(PKG_IS_DEFAULT_CHANNEL)
 
 # Generate package manifests.
-packagemanifests: manifests
+packagemanifests: kustomize manifests
   operator-sdk generate kustomize manifests -q
-  kustomize build config/manifests | operator-sdk generate packagemanifests -q --version $(VERSION) $(PKG_MAN_OPTS)
+  $(KUSTOMIZE) build config/manifests | operator-sdk generate packagemanifests -q --version $(VERSION) $(PKG_MAN_OPTS)
 ```
 
+**Note** If your project is an Ansible/Helm-based Operator, then the `packagemanifests` works at the same way. However, you need to remove `manifests` target since it do not exist for them. In this way, just replace `packagemanifests: kustomize manifests` with `packagemanifests: kustomize`. 
```

Comment:
Can you just add a separate rule instead of a note, ex:
````suggestion
```

For Go operator projects:
```make
# Generate package manifests.
packagemanifests: kustomize manifests
  operator-sdk generate kustomize manifests -q
  $(KUSTOMIZE) build config/manifests | operator-sdk generate packagemanifests -q --version $(VERSION) $(PKG_MAN_OPTS)
```

For other operator project types:
```make
# Generate package manifests.
packagemanifests: kustomize
  operator-sdk generate kustomize manifests -q
  $(KUSTOMIZE) build config/manifests | operator-sdk generate packagemanifests -q --version $(VERSION) $(PKG_MAN_OPTS)
```
````